roles/openshift_health_checker/openshift_checks/package_availability.py

```diff
@@ -36,7 +36,6 @@ def master_packages(rpm_prefix):
             "bash-completion",
             "cockpit-bridge",
             "cockpit-docker",
-            "cockpit-kubernetes",
             "cockpit-shell",
             "cockpit-ws",
```

Comment:
The cockpit packages were included a few releases ago because I found that a byo install would expect to install them at that time. May not be the case anymore? To some degree, nearly all packages are optional, but if they're missing from your repos it's a sign you're missing something and likely to break somewhere later (... if you're doing an RPM install). Honestly I expect to have to tweak the `package_availability` check once people are actually running it and seeing problems; just not sure what actual users with actual inventories are going to run into.

The cockpit-kubernetes RPM doesn't seem to be available in Origin and nothing seems to break without it so I removed it. I think it migrated to being run as a container only.